inc/common.php

```diff
@@ -34,3 +34,26 @@ function getRSSHeaders($url, $numHeaders) {
 			echo "<p>Nepodařilo se načíst RSS kanál.</p>";
 		}
 	}
+        
+        function isVisible($name){
+                foreach (explode(";", $_COOKIE["blocked"]) as $item) {
+                        if($item === $name){
+                                return false;
+                        }
+                }
+                return true;
+        }
+        
+        function updateCookies(){
```

Comment:
Nad tím jsem přemýšlel, ale chtěl jsem to udělat funkční i pro ty, kdo mají javascript vypnutý (NoScript) - pokud to tak někdo používá, určitě je to právě uživatel SeaMonkey. :)

Jinak názvy byly změněny v 390a2e2